content/en/docs/contribute/participate/issue-wrangler.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Issue Wranglers
+content_type: concept
+weight: 20
+---
+
+<!-- overview -->
+
+There are many issues that need triage, and in order to reduce our reliance on formal approvers or reviewers, we have introduced a new role to wrangle issues every week.The main responsibility of this role is to bridge the gap between organizational contributor and reviewer.
+This section covers the duties of a PR wrangler.
+
+<!-- body -->
+
+## Duties
+
+Each day in a week-long shift as Issue Wrangler:
+
+- Making sure the issue is worded and titled correctly to provide contributors with adequate information.
```

Comment:
```suggestion
- Triage and tag incoming issues daily. See [Triage and categorize issues](https://github.com/kubernetes/website/blob/main/docs/contribute/review/for-approvers/#triage-and-categorize-issues) for guidelines on how SIG Docs uses metadata.
```

Reply:
As an aside, we might want to remove this from the PR Wrangler file to avoid overlap. Thoughts?

Reply:
Totally agree!!